features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -317,6 +317,17 @@
         <OpenIDConnect>
             <IDTokenBuilder>org.wso2.carbon.identity.openidconnect.DefaultIDTokenBuilder</IDTokenBuilder>
             <SignatureAlgorithm>SHA256withRSA</SignatureAlgorithm>
+            <IDTokenEncryptionAlgorithm>RSA-OAEP</IDTokenEncryptionAlgorithm>
```

Comment:
Should not this go to OAuth section?

Reply:
I'm not very much familiar with that. The signature algorithm used to sign id tokens were placed here. I followed that. We can certainly move this to OAuth if that's the right place.

Reply:
I discussed with Darshana about this and we think it's best to keep this as it is since these are id token encryption algorithms. WDYT?